contrib/digitalocean/config.sh

```diff
@@ -2,18 +2,40 @@ export DEIS_TEST_DOMAIN="xip.io"
 export DO_TOKEN
 export DO_SSH_FINGERPRINT
 
-prompt "What Digital Ocean token should I use?" DO_TOKEN
-prompt "What Digital Ocean ssh fingerprint should I use?" DO_SSH_FINGERPRINT
+function check-do-token {
+  local token="${1:-}"
+
+  if [ ! -z ${token} ]; then
```

Comment:
can't we check for the presence of $1 here rather than assigning it a local variable?